From 678ab1d47c91e635aa5df5053a1be45a19d2bc08 Mon Sep 17 00:00:00 2001
From: Harald Judt <h.judt@gmx.at>
Date: Fri, 20 Dec 2013 18:36:49 +0100
Subject: Fix build with >=gcrypt-1.6.0.

gcry_thread_cbs struct seems to have changed, so build fails with
the error message "too many initializers". This patch fixes this.

Signed-off-by: Harald Judt <h.judt@gmx.at>
---
 src/crypto/Crypto.cpp | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/src/crypto/Crypto.cpp b/src/crypto/Crypto.cpp
index 6ad7fb0..bf6296d 100644
--- a/src/crypto/Crypto.cpp
+++ b/src/crypto/Crypto.cpp
@@ -49,13 +49,7 @@ static int gcry_qt_mutex_unlock(void** p_sys)
 
 static const struct gcry_thread_cbs gcry_threads_qt =
 {
-    GCRY_THREAD_OPTION_USER,
-    0,
-    gcry_qt_mutex_init,
-    gcry_qt_mutex_destroy,
-    gcry_qt_mutex_lock,
-    gcry_qt_mutex_unlock,
-    0, 0, 0, 0, 0, 0, 0, 0
+    GCRY_THREAD_OPTION_USER
 };
 
 Crypto::Crypto()
-- 
1.8.5.2

